example/App.tsx

```diff
@@ -182,11 +192,64 @@ class RewaredTest implements Test {
   }
 }
 
+class AdConsentTest implements Test {
+  getPath(): string {
+    return 'Consent Form';
+  }
+
+  getTestType(): TestType {
+    return TestType.Interactive;
+  }
+
+  render(onMount: (component: any) => void): React.ReactNode {
+    return (
+      <View style={styles.testSpacing} ref={onMount}>
+        <Button
+          title="Show Consent Form"
+          onPress={async () => {
+            const consentInfo = await AdsConsent.requestInfoUpdate({
+              debugGeography: AdsConsentDebugGeography.EEA,
+              testDeviceIdentifiers: [],
+            });
+
+            if (
+              consentInfo.isConsentFormAvailable &&
+              consentInfo.status === AdsConsentStatus.REQUIRED
+            ) {
+              await AdsConsent.showForm();
+            }
+          }}
+        />
+
+        <Text>
+          This test case will not work with the test App ID. You must configure
```

Comment:
even better - the specifics here will be a big help I'm sure, thanks for taking the time to put all the details in